{"name": "test_find_transaction_amount", "status": "broken", "statusDetails": {"message": "NameError: name 'transaction_amount' is not defined", "trace": "self = <tests.test_parabank.Test_Para_Bank object at 0x7f63c8e13e00>\n\n    def test_find_transaction_amount(self):\n        global transaction_id, transaction_amount\n        findtransactions = FindTransactions(self.driver)\n>       if transaction_amount:\nE       NameError: name 'transaction_amount' is not defined\n\ntests/test_parabank.py:147: NameError"}, "attachments": [{"name": "failed_test", "source": "48659b84-5772-432d-871b-dc0549e5b2f9-attachment.png", "type": "image/png"}], "start": 1746765988687, "stop": 1746765988688, "uuid": "ea0c2694-7885-447e-99f5-714fa0b78e10", "historyId": "e9e7bbd88cf4c34726641bac636c239a", "testCaseId": "e9e7bbd88cf4c34726641bac636c239a", "fullName": "tests.test_parabank.Test_Para_Bank#test_find_transaction_amount", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_parabank"}, {"name": "subSuite", "value": "Test_Para_Bank"}, {"name": "host", "value": "fv-az1689-517"}, {"name": "thread", "value": "3455-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_parabank"}]}